tests/models/test_detection.py

```diff
@@ -18,19 +19,54 @@ def test_fasterrcnn():
 
 
 def test_fasterrcnn_train(tmpdir):
-    model = FasterRCNN()
+    model = FasterRCNN(pretrained=False)
 
     train_dl = DataLoader(DummyDetectionDataset(), collate_fn=_collate_fn)
     valid_dl = DataLoader(DummyDetectionDataset(), collate_fn=_collate_fn)
 
-    trainer = pl.Trainer(fast_dev_run=True, default_root_dir=tmpdir)
+    trainer = pl.Trainer(
+        fast_dev_run=True, logger=False, checkpoint_callback=False, max_epochs=1, default_root_dir=tmpdir
+    )
```

Comment:
I guess we can drop `max_epochs=1`.

Reply:
Defensive programming in tests. :sweat_smile:

Reply:
Let me revert is :smile: